### PR TITLE
Update to Python 3.7 and simplify/cleanup

### PR DESCRIPTION
**Pull request by sqrrrl**
Update sample and dependencies to run on Python 3.7/App Engine standard. Resolve some linter issues as part of cleanup.

### CODE REVIEW

File: python/basic-bot/.gcloudignore [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# If you would like to upload your .git directory, .gitignore file or files
+# from your .gitignore file, remove the corresponding line
+# below:
+.git
+.gitignore
+
+# Python pycache:
+__pycache__/
+# Ignored by the build system
+/setup.cfg
```
Comment (asrivas) [inline python/basic-bot/.gcloudignore]: nit

File: python/basic-bot/README.md [hunk 2]
```diff
@@ -1,83 +1,84 @@
...
+messages from a room. The sample is built using Python 3 on Google App Engine,
 Standard Environment.
 
+## Deploy the sample
+
+  1. Follow the steps in [Setting Up Your Development Environment](https://cloud.google.com/appengine/docs/standard/python3/setting-up-environment)
+     to install Python and the Google Cloud SDK
+
+  1. Follow the steps in [Setting Up Your GCP Resources](https://cloud.google.com/appengine/docs/standard/python3/console/#create)
+     to create a project and emable App Engine.
```
Comment (asrivas) [inline python/basic-bot/README.md]: *enable

File: python/basic-bot/README.md [hunk 3]
```diff
@@ -1,83 +1,84 @@
...
+
+  1. Rum the following command to deploy the app:
```
Comment (asrivas) [inline python/basic-bot/README.md]: heh another nit

File: python/basic-bot/README.md [hunk 4]
```diff
@@ -1,83 +1,84 @@
...
+     the steps to enable the API in
+     [Publishing bots](https://developers.google.com/hangouts/chat/how-tos/bots-publish).
+  1. When configuring the bot on the **Configuration** tab on the
+     **Hangouts Chat API** page, enter the URL for the deployed version
+     of the bot into the **Bot URL** text box.
+
+
+## Interact with the bot
+
+Either add and @-mention the bot in a room or in a direct mention to engage with the bot.
```
Comment (asrivas) [inline python/basic-bot/README.md]: not picky if about using @-mention or @ mention, just consistency with line 22
looks like docs are inconsistent though and have @mention as well:
https://developers.google.com/s/results/hangouts/chat?q=mention

File: python/basic-bot/.gcloudignore (hunk 1 above)
Comment (sqrrrl) [reply python/basic-bot/.gcloudignore]: done

File: python/basic-bot/README.md (hunk 2 above)
Comment (sqrrrl) [reply python/basic-bot/README.md]: done

File: python/basic-bot/README.md (hunk 3 above)
Comment (sqrrrl) [reply python/basic-bot/README.md]: Fingers off by 1 :)

File: python/basic-bot/README.md (hunk 4 above)
Comment (sqrrrl) [reply python/basic-bot/README.md]: Updated both to be @mention(s).

**Pull request merged**